meilisearch-http/src/index/search.rs

```diff
@@ -78,19 +88,89 @@ impl Index {
         let mut documents = Vec::new();
         let fields_ids_map = self.fields_ids_map(&rtxn).unwrap();
 
-        let fields_to_display =
-            self.fields_to_display(&rtxn, query.attributes_to_retrieve, &fields_ids_map)?;
+        let displayed_ids: HashSet<FieldId> = fields_ids_map.iter().map(|(id, _)| id).collect();
```

Comment:
```suggestion
        let displayed_ids = sel.fields_to_display(&rtxn, query.attribute_to_retrieve, &fields_ids_map)?;
        let displayed_ids: HashSet<FieldId> = displayed_ids.iter().map(|(id, _)| id).collect();
```

Reply:
I think this should fix the bug @curquiza found about the `displayedAttributes` not being taken in account during the search